.github/actions/environment/action.yml

```diff
@@ -8,4 +8,13 @@ runs:
         echo TEST_TIMEOUT=310 | tee -a $GITHUB_ENV
         echo NO_OUTPUT_TIMEOUT=320 | tee -a $GITHUB_ENV
         echo PRESERVE_ENVVARS=REPLICATION_SYNC_TIMEOUT,TEST_TIMEOUT,NO_OUTPUT_TIMEOUT | tee -a $GITHUB_ENV
+        # This switching off swap command will not work as github step
+        # run from inside github 'container' tag. Also it will fail to
+        # run from it. So running it only outside of docker container.
+        # NOTE: To switch off swap from inside github 'container' tag
+        # additional memory flags should be added to its 'options' tag:
+        #   options: '--memory=<some value, like 7G> --memory-swap=<the same value as for memory option>'
+        if which docker && sudo ls /sbin/swapoff ; then
```

Comment:
I guess we don't need neither `sudo`, nor `ls`: just `[ -e /sbin/swapoff ]`.

Reply:
It was a trick here - I wanted to check that sudo is accepted for the user and in this way 'ls' will pass.

Reply:
Okay, since this is a trick, it should be briefly commented.

Reply:
Right, added comments on it.